.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.yarnpkg.com
```

Comment:
Another attempt to try and prevent local config from changing the registry specified by `yarn.lock` when we regenerate it for new versions. The previous `.yarnrc` did not work 🤔 . Either way I've manually corrected the `yarn.lock` again.